### ISSUE TITLE
Calling create on existing node does not return false

### ISSUE THREAD

**Issue by dougselph** [liebke/zookeeper-clj#14]
The [docstring](https://github.com/liebke/zookeeper-clj/blob/master/src/zookeeper.clj#L103-L106) for create indicates it will return false when the node already exists, but rather returns the path just as when the node did not previously exist.

user=> (use 'zookeeper)
nil
user=> (def c (connect "127.0.0.1"))
#'user/c
user=> (exists c "/a")
nil
user=> (create c "/a")
"/a"
user=> (exists c "/a")
{:dataLength 0, :numChildren 0, :pzxid 3591, :aversion 0, :ephemeralOwner 93275924363870216, :cversion 0, :ctime 1423281726221, :czxid 3591, :mzxid 3591, :version 0, :mtime 1423281726221}
user=> (create c "/a")
"/a"
user=>

**Comment by kazuhiro** [comment]
I can't reproduce your results. Can you say which version of Zookeeper do you use or create a specific unit test for this case?
```
user> (use 'zookeeper)
nil
user>  (def c (connect "127.0.0.1"))
#'user/c
user>  (exists c "/a")
nil
user> (create c "/a")
"/a"
user> (exists c "/a")
{:dataLength 0, :numChildren 0, :pzxid 2, :aversion 0, :ephemeralOwner 94230770646974464, :cversion 0, :ctime 1437847500119, :czxid 2, :mzxid 2, :version 0, :mtime 1437847500119}
user>  (create c "/a")
false
user> 
```

**Comment by neverfox** [comment]
This is still broken. I can keep calling create with the same path and I never get false. This is pretty serious as a lot of coordination techniques/recipes depend on getting failure feedback when trying to create a node.

**Comment by neverfox** [comment]
I have 0.9.4 in my classpath, but still get the same behavior as the OP.

**Comment by neverfox** [comment]
Indeed, I get the correct behavior from master. Can someone make a new release to clojars?
